src/angularfire2.spec.ts

```diff
@@ -15,80 +15,83 @@ import {
   AngularFire,
   FirebaseObjectObservable,
   FIREBASE_PROVIDERS,
-  FirebaseAuth,
-  FirebaseUrl,
-  FirebaseRef,
+  AngularFireAuth,
+  FirebaseConfig,
+  FirebaseApp,
   defaultFirebase,
-  FirebaseDatabase
+  FirebaseDatabase,
```

Comment:
Rename to AngularFireDatabase to be consistent with new naming scheme.

Add note about deprecating old name.